bento/src/main/java/com/yelp/android/bento/components/CarouselComponent.kt

```diff
@@ -58,6 +58,18 @@ open class CarouselComponent(
     }
 
     fun addAll(components: Collection<Component>) = components.forEach(::addComponent)
+
+    fun remove(index: Int) {
+        group.remove(index)
+    }
+
+    fun remove(component: Component) {
+        group.remove(component)
+    }
+
+    fun clear() {
+        group.clear()
+    }
```

Comment:
nit: would be nice to use expression notation for all these
e.g. `fun remove(component: Component) = group::remove`